test/modules/iframe/components/DrawTool.test.js

```diff
@@ -74,6 +74,107 @@ describe('DrawTool', () => {
 			});
 		});
 
+		describe('QueryParameters.EC_DRAW_TOOL is a comma-separated list of values', () => {
+			it('builds the full list of tools (from invalid toolName)', async () => {
+				const queryParam = new URLSearchParams(`${QueryParameters.EC_DRAW_TOOL}=fooBar`);
+				spyOn(environmentServiceMock, 'getQueryParams').and.returnValue(queryParam);
+				const element = await setup();
+
+				expect(element.shadowRoot.querySelectorAll('.draw-tool__button')).toHaveSize(3);
+				expect(element.shadowRoot.querySelector('#point-button')).toBeTruthy();
```

Comment:
Just a hint: For a long time we check the presence of an element with
```
expect(element.shadowRoot.querySelectorAll('#point-button')).toHaveSize(1)
```
cause when doing so we also ensure that there exist no doublets of this element